src/Main.java

```diff
@@ -0,0 +1,86 @@
+import java.util.Arrays;
+
+public class Main {
+    public static void main(String[] args){
+
+        // HomeTask 1
+        int[] arr2 = {1,4,3,2};
+        Arrays.sort(arr2);
```

Comment:
task specifies that you need to check if the array is sorted, there is no requirement to sort it additionally, only to check
so this line is unnecessary